omod/src/main/resources/apps/createUser_extension.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "${project.parent.groupId}.${project.parent.artifactId}.createUser.customFragments.userLocation",
```

Comment:
I thought we changed from "createUser"?

Reply:
Just updated @dkayiwa  
Sorry, I thought it was just ids specifically for this module.